vllm/config.py

```diff
@@ -64,6 +64,7 @@ def __init__(
         tokenizer_revision: Optional[str] = None,
         max_model_len: Optional[int] = None,
         quantization: Optional[str] = None,
+        device: Optional[str] = None,
```

Comment:
Consider adding a comment to ModelConfig like this?

device: The device to be used for the model. If None, we will default to use CPU as the device